src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/profiledetails/ProfileDetailsSyncService.kt

```diff
@@ -39,18 +37,27 @@ class ProfileDetailsSyncService(
     }
   }
 
-  suspend fun profileDetailsPhysicalAttributesChanged(event: ProfileDetailsChangedEvent) {
-    val profileType = event.profileType
-    val offenderNo = event.offenderIdDisplay
-    val bookingId = event.bookingId
+  suspend fun profileDetailsPhysicalAttributesChangedEvent(event: ProfileDetailsChangedEvent) =
+    profileDetailsPhysicalAttributesChanged(
+      profileType = event.profileType,
+      offenderNo = event.offenderIdDisplay,
+      bookingId = event.bookingId,
+    )
+
+  suspend fun profileDetailsPhysicalAttributesChanged(
+    profileType: String,
+    offenderNo: String,
+    bookingId: Long,
+    nomisProfileDetails: PrisonerProfileDetailsResponse? = null,
+  ) {
```

Comment:
Refactored this class so we can call the sync. without an event (i.e. directly from the move booking handler)